webapp/src/components/AuctionPage/AuctionPage.css

```diff
@@ -1,8 +1,16 @@
+.AuctionPage {
+  padding-top: 0px !important;
```

Comment:
I was going to ask you if doing `.Page > .AuctionPage { padding: 0; }` (without an `!important`) was better, but checking the rest of the pages we also do

![image](https://user-images.githubusercontent.com/692077/48900131-d221eb00-ee51-11e8-8c18-df400ec2bd8f.png)
and
![image](https://user-images.githubusercontent.com/692077/48900152-e1a13400-ee51-11e8-91af-b98e96c82059.png)

maybe it's a good idea to remove that padding and adding it when we need it, like in the /marketplace and profile page?

Reply:
mm i think from all the children of `<Page>` there's only 3 that don't need the padding, but the other +20 pages do:

<img width="377" alt="screen shot 2018-11-22 at 10 46 52 am" src="https://user-images.githubusercontent.com/2781777/48906724-2e7e0e00-ee44-11e8-8627-de484eddb9c2.png">